src/main/java/com/microsoft/aad/msal4j/SilentRequest.java

```diff
@@ -31,4 +33,20 @@
         application.getServiceBundle().getServerSideTelemetry().getCurrentRequest().forceRefresh(
                 parameters.forceRefresh());
     }
+
+    String getFullThumbprint(){
```

Comment:
This is repeated - maybe could be put in MsalRequest and reused in RefreshTokenRequest.

Reply:
Thumbprints of silent and refresh tokens requests are different